sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -692,6 +693,31 @@ public static CaseDataDto buildFromEventParticipant(EventParticipantDto eventPar
 		return caseData;
 	}
 
+	public static CaseDataDto buildFromTravelEntry(TravelEntryDto travelEntry, PersonDto person) {
+
+		CaseDataDto caseData = CaseDataDto.build(person.toReference(), travelEntry.getDisease());
+
+		caseData.setCaseOrigin(CaseOrigin.POINT_OF_ENTRY);
+		caseData.setResponsibleRegion(travelEntry.getResponsibleRegion());
+		caseData.setResponsibleDistrict(travelEntry.getResponsibleDistrict());
+		caseData.setResponsibleCommunity(travelEntry.getResponsibleCommunity());
+		caseData.setRegion(travelEntry.getResponsibleRegion());
+		caseData.setDistrict(travelEntry.getResponsibleDistrict());
+		caseData.setCommunity(travelEntry.getResponsibleCommunity());
+		caseData.setPointOfEntry(travelEntry.getPointOfEntry());
+		caseData.setPointOfEntryDetails(travelEntry.getPointOfEntryDetails());
+		caseData.setReportDate(travelEntry.getReportDate());
+
+		if (person.getPresentCondition() != null
```

Comment:
done.

Reply:
To be mentioned: The filling of case outcome data is analogously to the creation from EventParticipant (`CaseDataDto`, l.686ff)
```java
		if (person.getPresentCondition() != null && person.getPresentCondition().isDeceased() && eventDisease == person.getCauseOfDeathDisease()) {
			caseData.setOutcome(CaseOutcome.DECEASED);
			caseData.setOutcomeDate(new Date());
		}
```

Reply:
Yeah, that's an oversight for the creation from event participant then - seems like we didn't change this code when we refactored the case outcome <-> person death information context. Could you make the adjustment for the creation from event participants as well?

Reply:
Done.
Since the event date is not available here (loading the `EventDto` for the `EventReferenceDto` would be required, which is not possible within the `sormas-api`), `eventParticipant.getCreationDate()` is used as a fallback.